src/sagemaker/spark/processing.py

```diff
@@ -685,6 +718,72 @@ def __init__(
             network_config=network_config,
         )
 
+    def get_run_args(
+        self,
+        submit_app,
+        submit_py_files=None,
+        submit_jars=None,
+        submit_files=None,
+        inputs=None,
+        outputs=None,
+        arguments=None,
+        job_name=None,
+        configuration=None,
+        spark_event_logs_s3_uri=None,
+    ):
+        """Returns a RunArgs object.
+
+        This object contains the normalized inputs, outputs
+            and arguments needed when using a ``PySparkProcessor``
+            in a :class:`~sagemaker.workflow.steps.ProcessingStep`.
+
+        Args:
+            submit_app (str): Path (local or S3) to Python file to submit to Spark
```

Comment:
Can we just call this `code` to be consistent?

Reply:
I'd prefer this match the signature of the processor's `run()` method, which also takes `submit_app` which eventually becomes `code` in the ScriptProcessor. Users previously expressed confusion about this, they weren't sure what "code" meant in the context of a Spark processor.

Reply:
+1 using submit_app, but could you add reference to RunArgs.code here

Reply:
Will do